### PR TITLE
Milestone 2 merge to develop

### PR DESCRIPTION
**Pull request by eliasecasta**
In this milestone I created all the main components needed for the calculator, buttons, result display and the button panel. 

Please don't pay much attention to the CSS, this is of course not the final design version, right now is just for me to know the limits of the calc elements as well as testing the react-bootstrap plug-in.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,3 @@
-import './style.css';
+// import App from './components/App';
```
Comment (christianotieno) [inline src/App.js]: ## Remove unnecessary file
- [x] This project should only have one `App.js` file which you have rightfully placed in the `components` directory. DO you mind removing this one as it is redundant and of no use?

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import '../style.css';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+function App() {
```
Comment (christianotieno) [inline src/components/App.js]: ## Enforce the use of `ES6` syntax
- [x] Kindly change this function declaration into an arrow function.

File: src/components/Button.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { PropTypes } from 'prop-types';
+import Button from 'react-bootstrap/Button';
+
+function renderButton({ name = 'btn', color = 'light' }) {
```
Comment (christianotieno) [inline src/components/Button.js]: ## Enforce the use of `ES6` syntax
- [x] Kindly change this function declaration into an arrow function.

File: src/components/ButtonPanel.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import Button from './Button';
+
+function ButtonPanel() {
```
Comment (christianotieno) [inline src/components/ButtonPanel.js]: ## Enforce the use of `ES6` syntax
- [x] Kindly change this function declaration into an arrow function.

File: src/components/Display.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import { PropTypes } from 'prop-types';
+
+function Display({ result = '0' }) {
```
Comment (christianotieno) [inline src/components/Display.js:3]: ## Enforce the use of `ES6` syntax
- [ ] Kindly change this function declaration into an arrow function.

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @e71az 👋🏾 
Congratulations on implementing all the required components in this milestone. 
We hover do need to take care of some issues before proceeding 👇🏾 

### Changes
- Time for some cleaning
- [x] Rid the codebase of necessary files such as the `App.js` 
- [ ] Enforce the use of `ES6` syntax.

### Happy coding 💻 
#### Chrstian Otieno (CR)

File: src/App.js (hunk 1 above)
Comment (eliasecasta) [reply src/App.js]: Done, forgot to remove it after I made everything work haha!

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**